lib/merge.js

```diff
@@ -160,7 +160,10 @@ function getCommitMessage(mergeCommitMessage, pullRequest) {
   } else if (mergeCommitMessage === "pull-request-title-and-description") {
     return pullRequest.title + "\n\n" + pullRequest.body;
   } else {
-    throw new Error(`unknown commit message value: ${mergeCommitMessage}`);
+    ;['number', 'title', 'body'].forEach(prProp => {
```

Comment:
```suggestion
    ["number", "title", "body"].forEach(prProp => {
```

Possibly also change prProp to "property"

Reply:
To be honest, imo `property` is so much generic name, `prProp` sounds at least a bit more descriptive I think

Reply:
Fair enough!